pkg/rego/bind.go

```diff
@@ -0,0 +1,137 @@
+package rego
+
+import (
+	"encoding/json"
+	"fmt"
+	"reflect"
+
+	"github.com/open-policy-agent/opa/ast"
+)
+
+const tag = "rego"
+
+func Bind(src ast.Value, dst interface{}) error {
```

Comment:
This is a really neat idea! I'm sure this will be a big improvement over the JSON round trips.